wcomponents-core/src/main/java/com/github/bordertech/wcomponents/AbstractWComponent.java

```diff
@@ -1707,6 +1691,31 @@ public String getHtmlClass() {
 		return I18nUtilities.format(null, model.getHtmlClass());
 	}
 
+	/**
+	 * {@inheritDoc}
+	 */
+	@Override
+	public HashSet getHtmlClassList() {

```

Comment:
Where possible, it is good to use the interface "Set" instead of an implementation like "HashSet".

The name of the method is a little bit misleading....What if we call it getHtmlClasses().